addons/mail/static/tests/owl/component/composer_tests.js

```diff
@@ -108,6 +108,10 @@ QUnit.test('add an emoji', async function (assert) {
         "😊",
         "emoji should be inserted in the composer text input"
     );
+    // ensure popover is closed
+    await nextAnimationFrame();
+    await nextAnimationFrame();
+    await nextAnimationFrame();
```

Comment:
can't disable animation of popover instead?

Reply:
NFI, solution from @seb-odoo :D ... But maybe it could be fixed in a better way in composer task as it's not a problem due to this branch (just the test fails sometimes if you don't wait enough)